Dockerfile

```diff
@@ -1,18 +1,21 @@
-FROM ckan/ckan-dev:2.10.5
+FROM ckan/ckan-dev:2.10.5-py3.10
 # Inherit from here: https://github.com/okfn/docker-ckan/blob/master/ckan-dev/2.10/Dockerfile
 # And then from here: https://github.com/okfn/docker-ckan/blob/master/ckan-base/2.10/Dockerfile
 
-ENV CKAN_HOME /srv/app
-ENV CKAN_CONFIG /app/config
-ENV APP_DIR /app
+USER root
```

Comment:
a permission error is thrown without this which is weird because i thought docker uses `root` by default if not present